code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -140,6 +140,8 @@
 	item_state = "plasmaman_tank_belt"
 	slot_flags = SLOT_BELT
 	force = 5
+	volume = 3

```

Comment:
i set it to 3 because that is what the normal emergency tanks have. If you need more plasma, I've never had a round where I've been turned down when asking for it (as a plasmeme).